pmd-apex/src/main/resources/category/apex/bestpractices.xml

```diff
@@ -168,4 +168,43 @@ trigger Accounts on Account (before insert, before update, before delete, after
 ]]>
         </example>
     </rule>
+
+    <rule name="DebugsShouldUseLoggingLevel"
+          since="6.18.0"
+          language="apex"
+          message="Calls to System.debug should specify a logging level."
+          class="net.sourceforge.pmd.lang.apex.rule.ApexXPathRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_apex_bestpractices.html#debugsshoulduselogginglevel">
+       <description>
+The first parameter of System.debug, when using the signature with two parameters, is a LoggingLevel enum.
+
+Having the Logging Level specified provides a cleaner log, and improves readability of it.
+       </description>
+       <priority>5</priority>
+       <properties>
+          <property name="strictMode" type="Boolean" value="false" description="If true, mark statements that use the DEBUG enum of LoggingLevel."/>
+          <property name="version" value="2.0"/>
+          <property name="xpath">
+             <value>
+<![CDATA[
+//MethodCallExpression[@FullMethodName='System.debug'][count(*)=2]
```

Comment:
This is not using the `strictMode` property

Reply:
I wonder why tests are passing on my end then. Honestly I don't know why this expression works. I just copied the example from @rsoesemann. I think he might have a better way to handle this case.

Reply:
Maybe you didn't pull my commit ?